src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.abstracts.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
+public class Main {
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figureSupplier.getRandomFigure();
+            } else {
+                figureSupplier.getDefaultFigure();
+            }
```

Comment:
as per task requirement: 
"After generating the array, we need to display the entire list of objects that we have"